apps/src/blockly/addons/cdoFieldButton.js

```diff
@@ -4,12 +4,11 @@ const CORNER_RADIUS = 3;
 const INNER_HEIGHT = 16;
 
 export default class CdoFieldButton extends GoogleBlockly.Field {
-  constructor(title, opt_buttonHandler, opt_color, opt_changeHandler) {
+  constructor(title, opt_buttonHandler, _, opt_changeHandler) {
```

Comment:
A comment here could be helpful to understand why we don't need the third parameter.